x-pack/plugins/reporting/server/plugin.test.ts

```diff
@@ -88,7 +88,10 @@ describe('Reporting Plugin', () => {
   it('logs start issues', async () => {
     const plugin = new ReportingPlugin(initContext);
     // @ts-ignore overloading error logger
-    plugin.logger.error = jest.fn();
+    plugin.logger.error = jest.fn().mockImplementation((err) => {
+      // eslint-disable-next-line no-console
+      console.log(err);
+    });
```

Comment:
Is that used by the tests? If not, I would remove the console logging to avoid polluting the test logs.